modules/combined/examples/phase_field-mechanics/LandauPhaseTrans.i

```diff
@@ -0,0 +1,301 @@
+#
+# Martensitic transformation
+# Chemical driving force described by Landau Polynomial
+# Coupled with elasticity (Mechanics)
+#
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 100
+  ny = 100
+  nz = 0
+  xmin = 0
+  xmax = 100
+  ymin = 0
+  ymax = 100
+  zmin = 0
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[Variables]
+  [./eta1]
+    order = FIRST
+    family = LAGRANGE
+    [./InitialCondition]
+      type = RndBoundingBoxIC
```

Comment:
any reason you are not just using a `RandomIC`? The box seems to cover the entire domain anyways.